demos/vite-demo/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "vite-demo",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
```

Comment:
I see `index.js:2 Uncaught ReferenceError: exports is not defined    at index.js:2:23` when I run `pnpm i && pnpm dev`. Is there a step I'm missing?

Reply:
Yeah, I'm getting the same thing too as I'm trying to shift away from `npm` to using `pnpm` and referencing the `isograph` libraries locally...  I'll try to see what I can dig up online regarding that error message

Reply:
Oooh, that error is coming from `libs/isograph-disposable-types/dist`. So maybe I've set up the Vite stuff in a way that's incompatible with the locally built `libs`?

Reply:
Should work for you now, thanks to @PatrykWalach!